Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/CreateAndQueryChangeRequestV2/task.json

```diff
@@ -14,7 +14,7 @@
   ],
   "version": {
     "Major": 2,
-    "Minor": 161,
+    "Minor": 162,
```

Comment:
We keep it same as Sprint number (At least in tasks , lets keep the same numbering convention in extensions)